doc/man7/openssl-core.h.pod

```diff
@@ -10,7 +10,7 @@ openssl/core.h - OpenSSL Core types
 
 =head1 DESCRIPTION
 
-The <openssl/core.h> header file defines a number of public types that
+The file F<< <openssl/core.h> >> defines a number of public types that
```

Comment:
I don't quite understand this obsession with over-explaining what something is.  This should suffice:

```
F<< <openssl/core.h> >> defines ...
```

(btw, you can't even be sure it's a file.  With some installations, `openssl/core.h` might be a text library entry.  Nope, no joke)

Reply:
Because I want to start a sentence with an uppercase letter; the alternative is jarring for reading English.  And I can be sure it's a file in all places that openssl delivers.  If others do something different, they will have to fix the docs.  And therefore, having the "The file" tag will make it easier for them to find what to change. :)

Reply:
Yeahok. 

I took a minute to look around, and found examples that agree with you:

http://man7.org/linux/man-pages/man0/stdio.h.0p.html

However, I see a consistent 'The *\<stdio.h\>* header', so taking the cue from that, maybe this:

```
The F<< <openssl/core.h> >> header defines ...
```

Reply:
done.  fixup commit pushed.